src/Control/Lens/Fold.hs

```diff
@@ -424,8 +424,8 @@ takingWhile p l pafb = fmap runMagma . traverse (cosieve pafb) . runTakingWhile
 --
 -- Note: Many uses of this combinator will yield something that meets the types, but not the laws of a valid
 -- 'Traversal' or 'IndexedTraversal'. The 'Traversal' and 'IndexedTraversal' laws are only satisfied if the
--- new values you assign also pass the predicate! Otherwise subsequent traversals will visit fewer elements
--- and 'Traversal' fusion is not sound.
+-- new value you assign to the first target also does not pass the predicate! Otherwise subsequent traversals
```

Comment:
This would need an example expecially where changing the second target don't invalidate the laws. Hard to believe.